### ISSUE TITLE
Create a generic boxel container for the card and field

### ISSUE THREAD

**Issue by dnaidionov** [cardstack/cardstack#1353]
Create a generic container that would correctly move and transmute any card it contains. It should support following operations:
- transmute between different card sizes
  - between S, M, L sizes
  - between isolated and embedded
  - between field views in schema, layout, edit modes
  - between views in above modes and isolated
- move the card across the page
- drag and drop
- scale up/down
- display/hide platter where needed
- display/hide header where needed

Those operations should become part of card API. 

Test with several existing fields in the card host.

**Comment by jenweber** [comment]
These are my findings so far.

Since transitions between sibling routes are working nicely, I decided to see what it would take to go from a card in the library to a "view" card. I found that I was not really able to modify the CardRenderer component. Any changes such as adding an animated-beacon broke things.

I think we have overloaded the Animated containers and logic in that template, and need to rethink some of those animations. For example, adding an animated-beacon around the Embedded template causes an error of having "more than one beacon named X" or "undefined." This seems to be due to rerenders caused by the bright blue card header animation.

Step one is, simplify and clean up existing animations.

Step 2, make sure animated-beacon is available to use (I think it's critical here but will confirm).

Step 3 is to handle the pop & place/tweening from embedded to isolated. This could either be for cards with relationships to other cards, or also for the library to isolated.
